tests/ert_tests/ert3/storage/test_storage.py

```diff
@@ -134,117 +135,52 @@ def test_add_and_get_ensemble_record(tmpdir, raw_ensrec, ert_storage):
     ert.storage.init(workspace=tmpdir)
 
     ensrecord = ert.data.RecordCollection(records=raw_ensrec)
-    ert.storage.add_ensemble_record(
-        workspace=tmpdir,
+    future = ert.storage.transmit_record_collection(
+        record_coll=ensrecord,
         record_name="my_ensemble_record",
-        ensemble_record=ensrecord,
-    )
-
-    retrieved_ensrecord = ert.storage.get_ensemble_record(
-        workspace=tmpdir, record_name="my_ensemble_record"
-    )
-
-    assert ensrecord == retrieved_ensrecord
-
-
-@pytest.mark.requires_ert_storage
-@pytest.mark.parametrize(
-    "raw_ensrec",
-    (
-        [{"data": [i + 0.5, i + 1.1, i + 2.2]} for i in range(3)],
-        [{"data": {"a": i + 0.5, "b": i + 1.1, "c": i + 2.2}} for i in range(5)],
-        [{"data": {2: i + 0.5, 5: i + 1.1, 7: i + 2.2}} for i in range(2)],
-        [{"data": b"asdfkasjdhjflkjah21WE123TTDSG34f"}],
-    ),
-)
-def test_add_and_get_ensemble_parameter_record(tmpdir, raw_ensrec, ert_storage):
```

Comment:
Was this moved or deleted? If the latter, could you comment as to why?

Reply:
I was removed because it was testing the old way `RecordCollections` we added to storage, which has fundamentally changed and is covered by other tests